### PR TITLE
Fix bundle deploy with juju 1

### LINKED ISSUE #1912: Message incorrect when adding relation
With 2.1.9 on AWS I found ` /?store=cs:bundle/rails-example-single-6` and added the bundle to the canvas. The following message appeared.  `[Object object] `needs to be fleshed out.

screen shot 2016-08-10 at 13 26 43

### PR DESCRIPTION
**Pull request by hatched**
When deploying a bundle with Juju 1 modify the recordSet argument order to match the Juju 2 deploy call signature

Fixes #1912

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

Deploy the GUI via the charm with Juju 1 then deploy a charm and a bundle. All should work as expected

Upgrade your GUI in Juju 2 and deploy an application and a bundle. All should work as expected.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5577/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5578/
Test PASSed.

File: jujugui/static/gui/src/app/utils/bundle-importer.js [hunk 1]
```diff
@@ -428,6 +429,15 @@ YUI.add('bundle-importer', function(Y) {
       });
       this.fakebackend._loadCharm(record.args[0], {
         'success': function(charm) {
+          var charmSeries = charm.get('series');
+          // If we're using legacy Juju the bundlechangeslib does not return
+          // the series of the charms so we need to consider that and munge the
+          // record arguments accordingly. When we deicde to drop support for
```
Comment (bac) [inline jujugui/static/gui/src/app/utils/bundle-importer.js]: Typo: decide

**Comment by bac** [comment]
Did QA with juju 1. charm deployed fine. the bundle i picked deployed but one of the services had an install hook failure. Not the GUI's fault.

NO QA FOR JUJU 2. Someone else needs to do that.

**Comment by huwshimi** [comment]
Seems fine, but I had the same thing as @bac when I deployed a single charm (the bundles were fine), the difference was it was fine with Juju 1 and had errors Juju 2.

**Comment by hatched** [comment]
Thanks @bac @huwshimi :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**